### DISCUSSION TITLE
position of the floatinglocalvideo to top right?

### DISCUSSION THREAD

**Discussion by tj36** [Azure/communication-ui-library#3328] [Q&A]
Hello,

I would like to know how can I change the position of the floatinglocalvideo to top right when using video gallery. Also, I don't see any styles property to set the dimensions(height and width) for the floating local video. Please let me know if there is any property that can be used. Thanks.

```
<VideoGallery
{...videoGalleryProps}
localVideoViewOptions={{ scalingMode: 'Fit' }}
remoteVideoViewOptions={{ scalingMode: 'Fit' }}
layout="floatingLocalVideo"
>
```

**Comment by alkwa-msft** [comment]
Created this issue to track. 
https://github.com/Azure/communication-ui-library/issues/3342

**Comment by alkwa-msft** [comment]
Closing this discussion due to inactivity. Feel free to re-open if this is still an issue.
